cohort-evaluator-spark/pom.xml

```diff
@@ -21,7 +21,7 @@
 
     <properties>
         <image.tag>us.icr.io/vpc-dev-cohort-rns/cohort-evaluator-spark:a-p3.4.3-tekton-20210907202747-a57c643820323a5c76ddb8392aa64c3eab27ba45</image.tag>
-        <config.volume>cohort-config</config.volume>
```

Comment:
We had discussed that we were going to remove this entire example section from the pom and the documentation.